src/utilities/spacing/spacing.scss

```diff
@@ -17,11 +17,18 @@
     }
   }
 
+  /**
+    * 1. @TODO - Re-evaluate shorthand properties if they become fully supported
```

Comment:
The resolution here is to abandon shorthand properties for the moment, and substitute with `-start` and `-end` combinations, which appear to work across browsers without trying to use postcss-logical, which never gave output that worked in all cases.

Reply:
I'd like to see this comment revised in a few ways:

- [ ] Remove `@TODO`: There's nothing actionable here until something changes in browsers, so this is just extra noise.
- [ ] Remove the commented out shorthand values, they are also just noise.
- [ ] Revise the comment to be about _why_ we're using longhand values and not about _what_ we should do later. Example: "Ideally we could use the `margin-block` shorthand, but it is not as well supported in browsers."